components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineSession.kt

```diff
@@ -343,6 +345,7 @@ class SystemEngineSession(
         val webSettings = webView.settings
         webSettings.userAgentString = toggleDesktopUA(webSettings.userAgentString, enable)
         webSettings.useWideViewPort = enable
+        defaultSettings?.useWideViewPort = enable
```

Comment:
Hm won't this change the default for all sessions if desktop mode is toggled? Is this needed / if so can you add a comment?

Reply:
In other words, I don't think we should be writing default settings within a single session. So, wondering if you really need that.

Reply:
Makes sense to me